tests/zfs-tests/tests/functional/channel_program/lua_core/tst.encryption.zcp

```diff
@@ -0,0 +1,22 @@
+--
+-- This file and its contents are supplied under the terms of the
+-- Common Development and Distribution License ("CDDL"), version 1.0.
+-- You may only use this file in accordance with the terms of version
+-- 1.0 of the CDDL.
+--
+-- A full copy of the text of the CDDL should have accompanied this
+-- source.  A copy of the CDDL is also available via the Internet at
+-- http://www.illumos.org/license/CDDL.
+--
+
+--
+-- Copyright (c) 2017 by Delphix. All rights reserved.
```

Comment:
Small nit: not sure if you are related to Delphix, but at least let's update the date here.

Reply:
:+1: thanks, fixup'd!